driver-core/src/main/java/com/datastax/driver/core/Cluster.java

```diff
@@ -1610,7 +1635,7 @@ private CloseFuture close() {
                 configuration.getPolicies().getRetryPolicy().close();
                 reconnectionPolicy().close();
                 configuration.getPolicies().getAddressTranslator().close();
-                for (LatencyTracker tracker : trackers)
+                for (LatencyTracker tracker : latencyTrackers)
```

Comment:
Similar to `init()` and `onRegister`, shouldn't we also be calling `onUnregister` on `errorTrackers`?